targets/lenovo-x1/netvmExtraModules.nix

```diff
@@ -25,12 +25,15 @@
 
     internalNic = let
       vmNetworking = import ../../modules/microvm/virtualization/microvm/common/vm-networking.nix {
+        config = configH;
+        inherit lib;
         vmName = microvm.name;
         inherit (microvm) macAddress;
+        internalIP = 1;
       };
     in "${lib.head vmNetworking.networking.nat.internalInterfaces}";
 
-    elemen-vmIp = "192.168.100.253";
+    element-vmIp = "192.168.100.104";
```

Comment:
again, shouldnt this be taken from the [host.nix](https://github.com/tiiuae/ghaf/pull/584/files#diff-8180f7d1e010f8a2734247b6e1878f4e1df2e4b72459fc4d7af8e0215d3c7488R57).

As you can see there is already appearing to be a copy error as AppFlowy is .104 and element is .103

Reply:
Good catch. In this case IP is required, hostname won't work.
Getting it from unified config is in the future plans.
IP changed to correct one.